pkg/container/utils_linux.go

```diff
@@ -256,7 +256,7 @@ func Create(root, id string, config *configs.Config) (*RuncContainer, error) {
 	if cm.Exists() {
 		// pids, err := cm.GetAllPids()
 		// Reading PIDs can race with cgroups removal, so ignore ENOENT and ENODEV.
-		if err != nil && !errors.Is(err, os.ErrNotExist) && !errors.Is(err, unix.ENODEV) {
+		if !errors.Is(err, os.ErrNotExist) && !errors.Is(err, unix.ENODEV) {
```

Comment:
This will always return err.

Reply:
I think this was a golint suggestion... reverting let's see.